editor/plugins/script_editor_plugin.cpp

```diff
@@ -4025,6 +4015,18 @@ ScriptEditor::ScriptEditor(WindowWrapper *p_wrapper) {
 	script_forward->set_disabled(true);
 	script_forward->set_tooltip_text(TTR("Go to next edited document."));
 
+	if (p_wrapper->is_window_available()) {
+		menu_hb->add_child(memnew(VSeparator));
+
+		make_floating = memnew(ScreenSelect);
+		make_floating->set_flat(true);
+		make_floating->set_tooltip_text(TTR("Make the script editor floating. test"));
```

Comment:
```suggestion
		make_floating->set_tooltip_text(TTR("Make the script editor floating."));
```
👀

Reply:
Oups forgot to remove it 😅🤣. Just tested if I was at the right place. I'll remove it.